src/Files.Uwp/App.xaml.cs

```diff
@@ -230,362 +231,72 @@ static async Task OptionalTask(Task task, bool condition)
             }
         }
 
-        private void OnLeavingBackground(object sender, LeavingBackgroundEventArgs e)
-        {
-            DrivesManager?.ResumeDeviceWatcher();
-        }
-
         /// <summary>
         /// Invoked when the application is launched normally by the end user.  Other entry points
         /// will be used such as when the application is launched to open a specific file.
         /// </summary>
-        /// <param name="e">Details about the launch request and process.</param>
+        /// <param name="args">Details about the launch request and process.</param>
         protected override async void OnLaunched(LaunchActivatedEventArgs e)
         {
+            var activatedEventArgs = Microsoft.Windows.AppLifecycle.AppInstance.GetCurrent().GetActivatedEventArgs();
+
             await logWriter.InitializeAsync("debug.log");
-            Logger.Info($"App launched. Prelaunch: {e.PrelaunchActivated}");
+            Logger.Info($"App launched. Launch args type: {activatedEventArgs.Data.GetType().Name}");
 
             //start tracking app usage
-            SystemInformation.Instance.TrackAppUse(e);
-
-            bool canEnablePrelaunch = ApiInformation.IsMethodPresent("Windows.ApplicationModel.Core.CoreApplication", "EnablePrelaunch");
-
-            await EnsureSettingsAndConfigurationAreBootstrapped();
-            _ = InitializeAppComponentsAsync().ContinueWith(t => Logger.Warn(t.Exception, "Error during InitializeAppComponentsAsync()"), TaskContinuationOptions.OnlyOnFaulted);
-
-            var rootFrame = EnsureWindowIsInitialized();
-
-            if (e.PrelaunchActivated == false)
+            if (activatedEventArgs.Data is Windows.ApplicationModel.Activation.IActivatedEventArgs iaea)
             {
-                if (canEnablePrelaunch)
-                {
-                    TryEnablePrelaunch();
```

Comment:
I hear that Microsoft is planning to add pre-launch apis to WinAppSdk, something to keep in mind here in case we want to comment out the code instead of removing it.

Reply:
Interesting, though I'm not sure if it ever worked for Files. Code still available in Apo.xaml.cs.old.

Reply:
We've always had issues with it, but we know the code was good since we could manually pre-launch the app from visual studio. Either way, we can always copy this back in from an older commit.